Containerfile

```diff
@@ -0,0 +1,45 @@
+FROM stagex/busybox:sx2024.04.2@sha256:8cb9360041cd17e8df33c5cbc6c223875045c0c249254367ed7e0eb445720757 AS busybox
```

Comment:
Will need to make sure these are monitored via renovate.
May need upstream to use a better tagging scheme, or otherwise will need custom renovate config.

Reply:
Images are tagged with both the stagex version number and the actual version number of the source of the packaged software. We can use either.

Using matched release tags across all stagex items though does assure compatibility.

Reply:
considering out of scope for this PR, but happy to externally discuss adopting an alternative stagex tagging and vulnerability monitoring strategy for all turnkey projects. I would rather not deviate from existing norms in one repo though.

Reply:
> considering out of scope for this PR, but happy to externally discuss adopting an alternative stagex tagging and vulnerability monitoring strategy for all turnkey projects. I would rather not deviate from existing norms in one repo though.

The change results in us loosing the ability to respond and update in response to e.g. go standard library bugs. We need to come up with an answer as part of the PR